commercetools-models/src/main/java/io/sphere/sdk/orders/LineItemImportDraftBuilder.java

```diff
@@ -65,12 +68,16 @@ public LineItemImportDraftBuilder custom(@Nullable final CustomFieldsDraft custo
         return this;
     }
 
+    public LineItemImportDraftBuilder custom(@Nullable final ItemShippingDetailsDraft shippingDetails) {
```

Comment:
Shouldn't this builder method be called `shippingDetails`?

Reply:
😳  copy past mistake 😞